### DISCUSSION TITLE
Error in Testcase OL step LIS execution

### DISCUSSION THREAD

**Discussion by smf1959-mor** [NASA-LIS/LISF#1498] [Q&A] [answered]
I launched the following command **LISF-x86_64.AppImage -a LIS -- -f lis.config_noah36_ol** I have this error **Abort(1) on node 0 (rank 0 in comm 496): application called MPI_Abort(comm=0x84000000, 1) - process 0**
Could you indicate me how can i overcome this issue

**Comment by dmocko** [comment, marked as answer]
Hello,

Did this command generate a lislog file?

Please attach your lislog.0000 file.   There may be an error or warning message in there,
perhaps at the end.  Please also attach your lis.config_noah36_ol file.

**Reply by smf1959-mor** [reply]
thank you. The problem was **in missing forcing meteorological files in INPUT/NLDAS2.FORCING/2017**
